patches/chrome-common-importer-importer_bridge.h.patch

```diff
@@ -24,6 +25,9 @@ index 56d4259aba433df997802109cf5a8d28c03ebd13..26bb05d70c9674cab27112704b895797
 +
 +  virtual void UpdateStats(
```

Comment:
I think we can remove this patch. We're already patching in `BraveExternalProcessImporterBridge` into `ProfileImportImpl` so we can just cast bridge_ in `BraveImporter::StartImport`